LimitProcessorInterface.php

```diff
@@ -15,7 +15,8 @@ public function getRateLimit(Request $request);
 
     /**
      * @param Request $request
+     * @param callable $controller
      * @return string
      */
-    public function getRateLimitAlias(Request $request);
+    public function getRateLimitAlias(Request $request, callable $controller);
```

Comment:
only thing that I did not manager to solve in an elegant way is the `$controller` callable

Reply:
Will think about that case. 
Cause original intention were to make Rate Limit framework agnostic. 
With controller argument we are in situation, when interface is dictated by DI and symfony logic, that request and current controller can differ.